core/src/edu/asu/c3simulator/screens/MarketingScreen.java

```diff
@@ -0,0 +1,368 @@
+		mainTable.add(rightSection).expand().fill();
+		
+		return mainTable;
+	}
+	
+	/**
+	 * This method creates the the grid of products that the user can choose from and adds
+	 * them to a table.
+	 * 
+	 * @return The table that was created.
+	 */
+	private Table getProductGrid()
+	{
+		Table productGrid = new Table();
+		Product[] products = getProducts();
+		
+		for (int i = 0; i < products.length; i++)
+		{
+			Table newProduct = new Table();
+			Label productLabel = new Label(products[i].getName(), skin);
+			Image productIcon = products[i].getProductImage();
+			productIcon.addListener(new productIconListener(productLabel, products[i]));
+			
+			newProduct.add(productIcon).size(100f).row(); // TODO change to dynamic size
+			newProduct.add(productLabel);
+			
+			productGrid.add(newProduct).space(15);
+			
+			if ((i + 1) % 2 == 0 && i != products.length - 1)
+				productGrid.row();
+		}
+		
+		return productGrid;
+	}
+	
+	/**
+	 * This method creates the the grid of marketing types icons that the user can choose
+	 * from and adds them to a table.
+	 * 
+	 * @return The table that was created.
+	 */
+	private Table getMarketingGrid()
+	{
+		Table marketingGrid = new Table();
+		String[][] marketingIcons = {
+				{ "Internet Campaign", "images/placeholder-Computer-Icon.png" },
+				{ "Social Media", "images/placeholder-SocialMedia-icon.png" },
+				{ "Television and Radio", "images/placeholder--TV-icon.png" },
+				{ "Traditional Advertising", "images/placeholder-Billboard-icon.png" } };
+		
+		for (int i = 0; i < marketingIcons.length; i++)
+		{
+			Table newMarketingMethod = new Table();
+			Label marketingMethodLabel = new Label(marketingIcons[i][0], skin);
+			FileHandle iconLocation = Gdx.files.internal(marketingIcons[i][1]);
+			Texture iconTexture = new Texture(iconLocation);
+			Image marketingtIcon = new Image(iconTexture);;
+			marketingtIcon.addListener(new marketingIconListener(marketingMethodLabel,
+					marketingIcons[i][0]));
+			
+			newMarketingMethod.add(marketingtIcon).size(200f).row(); // TODO change to
+																		// dynamic size
+			newMarketingMethod.add(marketingMethodLabel);
+			
+			marketingGrid.add(newMarketingMethod).space(15);
+			
+			if ((i + 1) % 2 == 0 && i != marketingIcons.length - 1)
+				marketingGrid.row();
+		}
+		
+		return marketingGrid;
+	}
+	
+	/**
+	 * This method is a testing shell that creates different products and return them to
+	 * be displayed.
+	 * 
+	 * @return an array of the products created.
+	 */
+	private Product[] getProducts()
+	{
+		// TODO get products
+		Product[] products = { new Product("T-shirt 1", "default"),
+				new Product("T-shirt 2", "default"), new Product("Short 1", "default"),
+				new Product("Short 2", "default"), new Product("Hoodie 1", "default"),
+				new Product("Pants 1", "default"), new Product("Pants 2", "default"),
+				new Product("Pants 3", "default"), new Product("Pants 4", "default") };
+		return products;
+	}
```

Comment:
A more detailed TODO has been added but haven't been updated yet here for some reason.